subsys/bluetooth/host/monitor.c

```diff
@@ -74,18 +64,101 @@ static struct {
 	atomic_t other;
 } drops;
 
+static void drop_add(uint16_t opcode)
+{
+	switch (opcode) {
+	case BT_MONITOR_COMMAND_PKT:
+		atomic_inc(&drops.cmd);
+		break;
+	case BT_MONITOR_EVENT_PKT:
+		atomic_inc(&drops.evt);
+		break;
+	case BT_MONITOR_ACL_TX_PKT:
+		atomic_inc(&drops.acl_tx);
+		break;
+	case BT_MONITOR_ACL_RX_PKT:
+		atomic_inc(&drops.acl_rx);
+		break;
+#if defined(CONFIG_BT_BREDR)
+	case BT_MONITOR_SCO_TX_PKT:
+		atomic_inc(&drops.sco_tx);
+		break;
+	case BT_MONITOR_SCO_RX_PKT:
+		atomic_inc(&drops.sco_rx);
+		break;
+#endif
+	default:
+		atomic_inc(&drops.other);
+		break;
+	}
+}
+
+#if defined(CONFIG_BT_DEBUG_MONITOR_RTT)
+#include <SEGGER_RTT.h>
```

Comment:
I'm not fully sure if coding style allows this, but otherwise looks good

Reply:
I think this was already there, but just moved, right?

Reply:
I would prefer having it at the top of the file, yes